src/animations.ts

```diff
@@ -539,25 +561,58 @@ function getSampleQuestionComponents() {
 }
 
 
+function onSampleQuestionSectionIntersection(entries: IntersectionObserverEntry[], audioSourceToAnimationNameMap: Record<string, AnimationName>) {
+    if (entries[0].isIntersecting) {
+        // If a sample question animation is already playing, do not start a new animation.
+        if (SAMPLE_QUESTION_ANIMATIONS.indexOf(CURRENT_ANIMATION_INFO.name) >= 0) return
+
+        const sampleQuestionsContainer = $('.section-sample-questions .container-basic')
+        const firstAudioSource = sampleQuestionsContainer.find('div[data-element="url"]').first().text()
+        const animationName = audioSourceToAnimationNameMap[firstAudioSource]
```

Comment:
Are we sure `sampleQuestionsContainer` could be found, same for `'div[data-element="url"]'`, and that L 570 won't crash?

Reply:
Yep with jQuery, even if `sampleQuestionsContainer` doesn't point to an element, `firstAudioSource` will just be an empty string.

However this made me realize there's no need to keep `sampleQuestionsContainer` as a separate variable, will clean this up :)